lsp-iedit.el

```diff
@@ -68,5 +75,19 @@ See also `lsp-enable-symbol-highlighting'."
                                  (lsp--text-document-position-params))))
     (lsp-iedit--on-ranges (mapcar #'lsp:document-highlight-range highlights))))
 
+;;;###autoload
+(defun lsp-evil-multiedit-highlights ()
```

Comment:
Since we are adding non `iedit` functions/features to this file, I think we should rename it or move this code to another file.
IMO rename to something common between the 2 would be better though

Reply:
`evil-multiedit` uses `iedit` (it's just a "thin" layer above `iedit`), so in a way it is an `iedit` feature. But yep, we can consider what you're suggesting. My first version of this function had the name `lsp-iedit-evil-multiedit-highlights`, but I didn't like this name too much. But I can rename it back :)

Reply:
Got it, I think we can keep it as it is so :)